pyanaconda/modules/payload/payloads/dnf/packages/packages.py

```diff
@@ -355,6 +369,30 @@ def set_missing_ignored(self, missing_ignored):
         self.missing_ignored_changed.emit()
         log.debug("Ignore missing is set to %s.", missing_ignored)
 
+    @property
+    def broken_ignored(self):
+        """Ignore packages that have conflicts with other packages.
+
+        :rtype: bool
+        """
+        return self._broken_ignored
+
+    def set_broken_ignored(self, broken_ignored):
+        """Set if the packages that have conflicts with other packages should be ignored.
+
+        :param missing_ignored: True if broken packages should be ignored.
+        :type missing_ignored: bool
+        :raise: UnsupportedValueError if ignorebroken is disabled on this platform.
+        """
+        if not conf.payload.enable_ignore_broken_packages:
```

Comment:
It should be always ok to call the setter with `False`, no?

Reply:
Yes it should but on the other hand what's the point to set False something which is disabled and can't be True. I rather taken this approach, basically you can't use the API if the feature is disabled, it seems more clear to me.

Reply:
I can imagine an addon that would like to always disable this feature. With this approach, the addon will have to check `conf` first and then call the DBus method. I don't really like it. You should be allowed to disable unsupported features, because you might not really know whether they are unsupported in the current configuration.

Reply:
I don't agree.

The point is in general the default state for some future feature could be enabled not disabled. In that case, you want to be able to only enable the feature? Also when something is disabled it shouldn't be usable all the time, not just accepting something but not everything, that seems really confusing to me. If we really want to go this way, then we should probably instead of raising an exception just silently not assign the value. It will than behave consistently all the time but I really don't want to do that because than user of the API don't know if something really happened or not.

Checking the conf is not such a pain and also it is not needed. You can try to set the value and just check for exception, I don't see a problem in that.

Reply:
Ok, it is not really that important to me.